server/Makefile

```diff
@@ -39,6 +39,7 @@ install-dirs:
 
 install-python3-setup: install-bin install-lib
 	mkdir -p ${DESTDIR}/python3
+	${COPY} ../setup.py ../setup.cfg ../requirements.txt ${DESTDIR}
```

Comment:
Why do we need `setup.py` and `setup.cfg` in the source tree?  All we need to do is `pip3 install -r requirements.txt`, rigth?